OpenRA.Mods.Common/Widgets/ScrollItemWidget.cs

```diff
@@ -57,6 +57,7 @@ public static ScrollItemWidget Setup(ScrollItemWidget template, Func<bool> isSel
 			w.IsVisible = () => true;
 			w.IsSelected = isSelected;
 			w.OnClick = onClick;
+			w.ItemKey = Convert.ToString(Game.CosmeticRandom.Next());
```

Comment:
I wanted to use the menu item text as ItemKey but didn't know how to do it. This also works.